pep-0458.txt

```diff
@@ -1057,19 +1057,19 @@ References
 .. [11] https://mail.python.org/pipermail/distutils-sig/2013-May/020848.html
 .. [12] PEP 449, Removal of the PyPI Mirror Auto Discovery and Naming Scheme, Stufft
         http://www.python.org/dev/peps/pep-0449/
-.. [13] https://isis.poly.edu/~jcappos/papers/cappos_mirror_ccs_08.pdf
+.. [13] https://theupdateframework.github.io/papers/attacks-on-package-managers-ccs2008.pdf
 .. [14] https://mail.python.org/pipermail/distutils-sig/2013-September/022755.html
 .. [15] https://pypi.python.org/security
-.. [16] https://github.com/theupdateframework/tuf/blob/develop/docs/tuf-spec.txt
+.. [16] https://github.com/theupdateframework/specification/blob/master/tuf-spec.md
 .. [17] PEP 426, Metadata for Python Software Packages 2.0, Coghlan, Holth, Stufft
         http://www.python.org/dev/peps/pep-0426/
 .. [18] https://en.wikipedia.org/wiki/Continuous_delivery
 .. [19] https://mail.python.org/pipermail/distutils-sig/2013-August/022154.html
 .. [20] https://en.wikipedia.org/wiki/RSA_%28algorithm%29
 .. [21] https://en.wikipedia.org/wiki/Key-recovery_attack
-.. [22] http://csrc.nist.gov/publications/nistpubs/800-57/SP800-57-Part1.pdf
+.. [22] https://doi.org/10.6028/NIST.SP.800-57pt1r4
 .. [23] https://www.openssl.org/
-.. [24] https://pypi.python.org/pypi/pycrypto
+.. [24] https://github.com/pyca/cryptography
```

Comment:
This used to point to pycrypto, which is not used anymore in the TUF reference implementation. Instead it uses [cryptography](https://github.com/pyca/cryptography) and [PyNaCl](https://github.com/pyca/pynacl), both optionally, and [ed25519](https://github.com/pyca/ed25519) for a minimal pure Python installation. On a side note, the TUF team is also working on support for [OpenPGP](https://tools.ietf.org/html/rfc4880) with [gnupg](https://gnupg.org/) ([#174](https://github.com/secure-systems-lab/securesystemslib/pull/174)), HSM signing with [PyKCS11](https://github.com/LudovicRousseau/PyKCS11) ([#170](https://github.com/secure-systems-lab/securesystemslib/pull/170)), and [SPHINCS +](https://sphincs.org/) with [PySPX](https://github.com/sphincs/pyspx) ([#169](https://github.com/secure-systems-lab/securesystemslib/pull/169)).

Let me know if any of this information should be incorporated in the PEP.